src/nrtk_explorer/library/nrtk_transforms.yaml

```diff
@@ -1,13 +1,143 @@
-nrtk_blur_2:
+nrtk_pil_brightness:
+  perturber: nrtk.impls.perturb_image.generic.PIL.enhance.BrightnessPerturber
+  exec_default_args: []
+  description:
+    factor:
+      _path: [factor]
+      type: float
+      label: Factor
+
+nrtk_pil_color:
+  perturber: nrtk.impls.perturb_image.generic.PIL.enhance.ColorPerturber
+  exec_default_args: []
+  description:
+    factor:
+      _path: [factor]
+      type: float
+      label: Factor
+
+nrtk_pil_contrast:
+  perturber: nrtk.impls.perturb_image.generic.PIL.enhance.ContrastPerturber
+  exec_default_args: []
+  description:
+    factor:
+      _path: [factor]
+      type: float
+      label: Factor
+
+nrtk_pil_sharpness:
+  perturber: nrtk.impls.perturb_image.generic.PIL.enhance.SharpnessPerturber
+  exec_default_args: []
+  description:
+    factor:
+      _path: [factor]
+      type: float
+      label: Factor
+
+nrtk_cv2_average_blur:
+  perturber: nrtk.impls.perturb_image.generic.cv2.blur.AverageBlurPerturber
+  description:
+    ksize:
+      _path: [ksize]
+      type: integer
+      label: Kernel Size
+
+nrtk_cv2_gauss_blur:
   perturber: nrtk.impls.perturb_image.generic.cv2.blur.GaussianBlurPerturber
   description:
     ksize:
       _path: [ksize]
       type: integer
       label: Kernel Size
-      default: 1
```

Comment:
any reason you remove the default?

Reply:
The default gets set automatically by the nrtk perturber at construction time, and we also fetch the current value at run time, so there was no need to have an internal default value for these classes